fdbclient/NativeAPI.actor.cpp

```diff
@@ -3027,29 +3032,25 @@ ACTOR Future<Void> readVersionBatcher( DatabaseContext *cx, FutureStream< std::p
 				else if (!timeout.isValid())
 					timeout = delay(batchTime, TaskPriority::ProxyGetConsistentReadVersion);
 			}
-			when(wait(timeout.isValid() ? timeout : Never())) {
-				send_batch = true;
-			}
+			when(wait(timeout.isValid() ? timeout : Never())) { send_batch = true; }
 			// dynamic batching monitors reply latencies
-			when(double reply_latency = waitNext(replyTimes.getFuture())){
+			when(double reply_latency = waitNext(replyTimes.getFuture())) {
 				double target_latency = reply_latency * 0.5;
 				batchTime = min(0.1 * target_latency + 0.9 * batchTime, CLIENT_KNOBS->GRV_BATCH_TIMEOUT);
 			}
-			when(wait(collection)){} // for errors
+			when(wait(collection)) {} // for errors
 		}
 		if (send_batch) {
 			int count = requests.size();
 			ASSERT(count);
-
 			// dynamic batching
 			Promise<GetReadVersionReply> GRVReply;
 			requests.push_back(GRVReply);
-			addActor.send(timeReply(GRVReply.getFuture(), replyTimes));
+			addActor.send(timeReplyIgnoreError(GRVReply.getFuture(), replyTimes));
```

Comment:
I end up keeping it so we have something explicit to look for in cases where txns with batch priority seem to get stuck.

Reply:
I think this question still applies because `timeReplyIgnoreError` will be logging errors. If that error is going to log for every failed batch, then reverting to the formulation I proposed above is probably the cleanest solution. Or, if you are wanting to log errors besides the throttling error, we would probably need to introduce some filtering.

Reply:
Oh right. It's been a while and I forgot I also log errors there. Good catch. I think we can do it without any logging for now and in future, if necessary, add the loggings in separate PRs. Sorry about the back and forth.

Reply:
Instead of `errorOr()`, `ready()` is better since the prior produces a type that needs to be map'd out.